docs/integrations/mirroring.md

```diff
@@ -261,5 +261,6 @@ The following incident fields must be configured either in the integration or in
 
 * When using a custom mapper, make sure it has the required mirroring incident fields (dbotMirrorDirection, dbotMirrorId and dbotMirrorInstance), more information can be found under the "Incident Fields on a Cortex XSOAR Incident" section.
 * The mapper the incident goes through upon creation configures whether the incident will be mirrored and in which direction. 
+* When switching between different mappers, that have different incident fields, for the mirroring to work with the new incident fields, you should delete the old incidents and re-fetch them with the newly configured mapper.
```

Comment:
Recommend:
When switching between different mappers with different incident fields, for mirroring to work with the new incident fields, delete the old incidents and re-fetch them with the new mapper.